### PR TITLE
Add Loading feature button

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Screenshot_1697631135

Screenshot_1697631132
Screenshot_1697631128
Screenshot_1697631125

### CODE REVIEW

**Review by rthanga1 — CHANGES_REQUESTED** [review]

File: DemoApp/DemoApp/Views/ButtonSamplePage.xaml [hunk 1]
```diff
@@ -114,6 +116,7 @@
                                  HorizontalOptions="{Binding FullWidthAlignment}"
                                  ButtonColor="Tertiary"
                                  Size="{Binding SelectedFontSize}"
+                                 IsLoading="{Binding IsLoading}"
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/ButtonSamplePage.xaml:119]: Yes, changed it

File: DemoApp/DemoApp/Views/ButtonSamplePage.xaml [hunk 2]
```diff
@@ -202,6 +208,14 @@
...
                 IsToggled="{Binding IsDisabled}"
                 Toggled="IsDisabled_Toggled" />
+        <Label Grid.Column="2"
+               Text="IsLoading"
+               HorizontalOptions="Center"
+               VerticalOptions="Center" />
+        <Switch Grid.Column="3"
+                x:Name="isLoading"
+                IsToggled="{Binding IsLoading}"
+                Toggled="isLoading_Toggled" />
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/ButtonSamplePage.xaml:218]: Yes, removed IsToggled Property.

File: Trimble.Modus.Components/Controls/Button/TMButton.xaml.cs [hunk 3]
```diff
@@ -169,16 +179,38 @@ private static void OnIsDisabledChanged(BindableObject bindable, object oldValue
...
             {
-                UpdateButtonStyle(button);
-                button.Opacity = 1;
+                // Hide the first column by setting its width to 0.
+               button.buttonStackLayout.ColumnDefinitions[0].Width = new GridLength(0);
             }
+            disableButton(isLoading, button);           
+        }
+    }
+    private static void disableButton(bool toggleDisable,TMButton button)
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply Trimble.Modus.Components/Controls/Button/TMButton.xaml.cs:203]: Yes, changed

File: DemoApp/DemoApp/Views/ButtonSamplePage.xaml [hunk 4]
```diff
@@ -202,6 +208,14 @@
                 x:Name="isDisabled"
                 IsToggled="{Binding IsDisabled}"
                 Toggled="IsDisabled_Toggled" />
+        <Label Grid.Column="2"
+               Text="IsLoading"
+               HorizontalOptions="Center"
+               VerticalOptions="Center" />
+        <Switch Grid.Column="3"
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/ButtonSamplePage.xaml:215]: Yes, changed

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**